src/components/form/HdRange.vue

```diff
@@ -0,0 +1,320 @@
+<template>
+  <div
+    :class="fieldClasses"
+    class="range field"
+  >
+    <input type="range"
+      ref="input"
+      v-model="currentValue"
+      :id="name"
+      :name="name"
+      :required="required"
+      :disabled="disabled"
+      :min="minValue"
+      :max="maxValue"
+      :step="rangeStep"
+      @input="updateRangeDecoration"
+      @change="handleChange"
```

Comment:
Now that we are using `v-model`, we don't have to handle these events manually. We can remove them.

Reply:
It does not work without those!